.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -11,67 +11,71 @@ assignees: ''
 
-1. **Identify Pull Requests**: Review any pull requests (PRs) that need to be backported to the release branch.
-2. **Backporting Process**: For PRs merged into the `main` branch that need to be applied to the release branch (e.g., `release/2.1`), comment on the merged PR with `/backport to release/2.1`. This comment will trigger a GitHub Action to create a new PR with the same changes for the `release/2.1` branch.
+1. **Identify Pull Requests**: Review any pull requests (PRs) that need to be backported to the release branch. These should be PRs that contain strictly bug fixes and nothing else.
+2. **Backport Pull Requests**: For PRs merged into the `main` branch that need to be applied to the release branch (e.g., `release/2.1`), comment on the merged PR with `/backport to release/2.1`. This comment will trigger a GitHub Action to create a new PR with the same changes for the `release/2.1` branch.
 3. **Merge PRs**: Once all necessary PRs are created, merge them into the `release/2.1` branch.
 
 ## Step 2: Code and Documentation Updates
 
-### Create Pull Request:
-
-  - [ ] From the release branch (e.g., `release/2.1`), create a new temporary branch for your release (e.g., `release-notes/2.1.1`).
-  - [ ] Update version references in the documentation. Refer to [this PR](https://github.com/OrchardCMS/OrchardCore/pull/17065/files) for an example. Version Updates Checklist:
-     - **Update `OrchardCore.Commons.props`**: Set `<VersionSuffix></VersionSuffix>` to the new version you're preparing for release.
-     - **Update Module Versions**: Modify `src/OrchardCore/OrchardCore.Abstractions/Modules/Manifest/ManifestConstants.cs` to reflect the new version.
-     - **Release Notes**: Finalize the release notes in the documentation, including:
-       - Highlights and goals of the release.
-       - Prerequisites for running the new version.
-       - Upgrade steps and any breaking changes.
-     - **Update Documentation Navigation**: Add the release notes page to `mkdocs.yml` navigation and remove it from `not_in_nav`.
-     - **Version Mentions**: Update all references to the new version throughout the documentation, including:
-       - [Status in the root README](https://docs.orchardcore.net/en/latest/#status)
-       - CLI templates and commands.
-       - Relevant guides, such as the [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/) guide.
-  - [ ] Create a **Documentation PR** titled "Release with the new version number" (e.g., `Release 2.1.1`) from the documentation branch (e.g., `release-notes/2.1.1`) into the release branch (e.g., `release/2.1`)
-  - [ ] Merge the Documentation PR.
-  - [ ] In GitHub, manually run the `Preview - CI` workflow on your branch (NOT `main`). This will release a new preview version on CloudSmith for testing.
+- [ ] From the release branch (e.g., `release/2.1`), create a new temporary branch for the version (e.g., `release/2.1.1`).
```

Comment:
```suggestion
- [ ] From the release branch (e.g., `release/2.1`), create a new temporary branch for the version (e.g., `release-notes/2.1.1`).
```
This was purposely set to release-notes to distinguish the docs branches from an actual release branch.

Reply:
But it's not just about the release notes. There are other preparations as well, that the guide included already (changing the version numbers in the code, updating the guides), that need to happen in some branch and shouldn't happen directly in the release one.

Reply:
yes but these are all administrative changes. I am fine with `release-notes`, if you want to change it, please change it to something other than `release` this way we don't confuse it with an actual release branch.

Reply:
`release-prep`?

Reply:
Sure

Reply:
That's in the template now.